docs/conf.py

```diff
@@ -16,17 +16,25 @@
 
 
 # -- Project information -----------------------------------------------------
+import os
+
 import rubrix as rb
 
 project = "Rubrix"
 copyright = "2021, Recognai"
 author = "Recognai"
 
-# The full version, including alpha/beta/rc tags
-release = rb.__version__
+# Normally the full version, including alpha/beta/rc tags.
+# But since rtd changes some files during the build process, the scm version becomes dirty,
+# so we just want to display the major and minor version in the tab title of the browser.
+release = ".".join(rb.__version__.split(".")[0:2])
```

Comment:
Could this brings conflicts in documentation if we publish a version patch (0.6.1, 0.6.2,.. for example)?

Reply:
I think the only conflict will be in the title of the tab, for both versions (0.6.1, 0.6.2) it will state 0.6.
The rest should not be affected. Haven't tested it though.

Reply:
Ok. We can hide patch version for documentation